plugins/ms-vscode.go/0.9.2/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM eclipse/che-theia-endpoint-runtime:nightly
```

Comment:
hello, FYI missing copyright header

Reply:
question: Dockerfile should go in che-theia, no ?

Reply:
you are right